sdk/identity/azure-identity/README.md

```diff
@@ -346,6 +346,8 @@ Credentials can be chained together to be tried in turn until one succeeds using
 
 ### Username and password
 
+> **Warning**: Username and password authentication doesn't support multifactor authentication (MFA) and is **deprecated**. For more details, see [Planning for mandatory MFA](https://aka.ms/azsdk/identity/mfa).
```

Comment:
Remove this entire "Username and password" section from the README. We don't want to draw attention to a flow that's deprecated.